code/modules/mining/equipment_locker.dm

```diff
@@ -557,9 +557,18 @@
 
 /obj/item/weapon/resonator/attack_self(mob/user)
 	if(burst_time == 50)
+		burst_time = 10
+		user << "<span class='info'>You set the resonator's fields to detonate after 1 second.</span>"
```

Comment:
a 1 second delay is super short, and with 5 fields max, it'd make the resonator OP (and I'm not even taking into account the buff to damage). :-1:
The resonator (and each of the mining tools) has a specific use: it's very slow at digging straight lines and at fighting mobs (but you don't have to go melee which is a plus), but it's good at digging asteroid turfs that are next to each other. Basically, it's good to dig a circle around you, so if you follow the caves of the asteroid instead of mining in a straight line you can achieve a good speed.